src/PokemonsList.tsx

```diff
@@ -14,6 +14,7 @@ export function PokemonsList(): JSX.Element {
   const [pagesToUpload, setPagesToUpload] = useState<number[]>([]);
   const [pages, setPages] = useState<PokeListResponse[]>([]);
   const bottomRef = useRef<HTMLDivElement>(null);
+  const [selectedPokemons, setSelectedPokemons] = useState<string[]>([]);
```

Comment:
PokemonList should be React managed component.
1) It is not possible to know what pokemons are selected outside the component
2) It is not possible to set selected pokemons